src/test/groovy/graphql/GraphQLTest.groovy

```diff
@@ -232,6 +232,33 @@ class GraphQLTest extends Specification {
         result.errors.size() == 0
     }
 
+    def "document with two operations executes them both in sequence"() {
+        given:
+
+        GraphQLSchema schema = newSchema().query(
+                newObject()
+                        .name("RootQueryType")
+                        .field(newFieldDefinition().name("field1").type(GraphQLString).dataFetcher(new StaticDataFetcher("value1")))
+                        .field(newFieldDefinition().name("field2").type(GraphQLString).dataFetcher(new StaticDataFetcher("value2")))
+        )
+                .build()
+
+        def query = """
+        query Query1 { field1 }
+        query Query2 { field2 }
+        """
+
+        def expected = [field2: 'value2']
+
+        when:
+        def executionInput = newExecutionInput().query(query).operationNames('Query1', 'Query2').context(null).variables([:])
+        def result = GraphQL.newGraphQL(schema).build().execute(executionInput)
+
+        then:
+        result.data == expected
+        result.errors.size() == 0
+    }
+
```

Comment:
This is the worse outcome I think - that is the result is thrown away.  

This is the biggest design blocker as discussed on https://github.com/graphql-java/graphql-java/issues/806

Reply:
Yeah I punted on figuring out what to do here, opting for the user to configure how the results should be accumulated (maybe with some common ones available in the lib, such as nesting data under the operation name).